app/src/main/java/org/mozilla/tv/firefox/FirefoxApplication.kt

```diff
@@ -55,6 +55,10 @@ open class FirefoxApplication : LocaleAwareApplication() {
         }
     }
 
+    private fun initServiceLocator() {
```

Comment:
nit: Do you think this method is necessary?

Reply:
Not anymore.  I'll pull it out.